packages/experiments/src/components/FolderCover/FolderCover.tsx

```diff
@@ -136,13 +125,11 @@ export function getFolderCoverLayout(element: JSX.Element): IFolderCoverLayout {
 }
 
 function getFolderCoverLayoutFromProps(folderCoverProps: IFolderCoverProps): IFolderCoverLayout {
-  const { folderCoverSize = 'large', isFluent } = folderCoverProps;
+  const { folderCoverSize = 'large' } = folderCoverProps;
 
   const contentSize = { ...SIZES[folderCoverSize] };
 
-  if (isFluent) {
-    contentSize.height -= 8;
-  }
+  contentSize.height -= 8;
```

Comment:
I assume this operation can be made static now that it is not gated by a conditional.

Reply:
fixed with latest push